code/modules/clothing/shoes/miscellaneous.dm

```diff
@@ -198,7 +198,7 @@
 	jumping = TRUE
 	playsound(src.loc, 'sound/effects/stealthoff.ogg', 50, 1, 1)
 	usr.visible_message("<span class='warning'>[usr] dashes foward into the air!</span>")
-	usr.throw_at(target, jumpdistance, 1, spin=0, diagonals_first = 1, callback = CALLBACK(src, .proc/hop_end))
+	usr.throw_at(target, jumpdistance, 3, spin=0, diagonals_first = 1, callback = CALLBACK(src, .proc/hop_end))
```

Comment:
@ExcessiveUseOfCobblestone This is throwspeed and IIRC this will be faster? Sure this is slower?
@Supermichael777 mobs are based on world.time so unless you want to raise server tickrate or whatever or make them default slower that's not gonna happen. or you can make us use pixel movement but goodluck lmao